auth-web/src/assets/scss/theme.scss

```diff
@@ -73,6 +74,9 @@ $BCgovInputBG: #ffffff;
 // Font Colors
 $BCgovFontColorInverted: #ffffff;
 
+// Warning Colors
+$BCgovPausedWarning: #EF6A01;
```

Comment:
This'll be the background color of the new badge:
![paused bg color](https://github.com/bcgov/sbc-auth/assets/122301442/975bd0c1-d494-4d75-bf58-d0432d7b3b04)

Reply:
I suggest a more generic name so it can be reused.

Cause this would look weird: `<icon name="pumpkin" color="paused-warning" />" 😁